large-borrows-governance/src/autotask.spec.js

```diff
@@ -0,0 +1,155 @@
+const ethers = require('ethers');
+
+// DEFENDER FORTA SENTINEL AUTOTASK TESTING //
+
+// grab the existing keys before loading new content from the .env file
+const existingKeys = Object.keys(process.env);
+require('dotenv').config();
+
+// now filter out all of the existing keys from what is currently in the process.env Object
+const newKeys = Object.keys(process.env).filter((key) => existingKeys.indexOf(key) === -1);
```

Comment:
```suggestion
const newKeys = Object.keys(process.env).filter((key) => !existingKeys.includes(key));
```
Nit. Includes does what you want to do.